Libraries/Microsoft.Teams.Plugins/Microsoft.Teams.Plugins.AspNetCore.BotBuilder/Extensions/HostApplicationBuilder.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.AspNetCore.Mvc.ApplicationParts;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Teams.Plugins.AspNetCore.Controllers;
+
+namespace Microsoft.Teams.Plugins.AspNetCore.Extensions;
+
+public static class HostApplicationBuilderExtensions
+{
+    public static IHostApplicationBuilder AddBotBuilder(this IHostApplicationBuilder builder)
+    {
+        builder.Services.AddControllers().ConfigureApplicationPartManager((apm) => {
+            apm.FeatureProviders.Add(new RemoveDefaultMessageController());
```

Comment:
Clean way to remove the message controller added by `IHostApplicationBuilder.AddTeams()`. It's run after all the controllers are discovered so order doesn't matter.